photutils/aperture/tests/test_circle.py

```diff
@@ -6,29 +6,76 @@
 from astropy.coordinates import SkyCoord
 import astropy.units as u
 import numpy as np
+import pytest
 
 from .test_aperture_common import BaseTestAperture
-from ..circle import (CircularAperture, CircularAnnulus, SkyCircularAperture,
-                      SkyCircularAnnulus)
+from ..circle import (
```

Comment:
please remove the code reformatting commit when rebasing

Reply:
Agreed.